### PR TITLE
feat: add `constants/float64/max-safe-nth-tribonacci`

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   adds `constants/float64/max-safe-nth-tribonacci`, which would help to complete the TODO at: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/tribonacci/src/main.c#L21

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+<!-- eslint-disable id-length -->
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var FLOAT64_MAX_SAFE_NTH_TRIBONACCI = require( '@stdlib/constants/float64/max-safe-nth-tribonacci' );
+
+var v;
+var i;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md]: @gunjjoshi Mind moving this declarations down closer to their usage just before the loop at L92?

File: lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/examples/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var FLOAT64_MAX_SAFE_NTH_TRIBONACCI = require( './../lib' ); // eslint-disable-line id-length
+
+var v;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/examples/index.js]: @gunjjoshi Same comment as README.

**Review by kgryte — CHANGES_REQUESTED** [review]
Apart from two small nitpicks, this LGTM.

File: lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md (hunk 1 above)
Comment (gunjjoshi) [reply lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md]: Done, thanks!

File: lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        a = b;
+        b = c;
+        c = d;
+    }
+    return c;
+}
+
+var v;
+var i;
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/README.md]:
```suggestion
```

File: lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/examples/index.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+		a = b;
+		b = c;
+		c = d;
+	}
+	return c;
+}
+
+var v;
+var i;
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/constants/float64/max-safe-nth-tribonacci/examples/index.js]:
```suggestion
```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Pull request merged**